services/compute/virtualmachine/virtualmachine.go

```diff
@@ -201,6 +201,7 @@ func (c *client) getWssdVirtualMachineHardwareConfiguration(vm *compute.VirtualM
 		VMSize:                     sizeType,
 		CustomSize:                 customSize,
 		DynamicMemoryConfiguration: dynMemConfig,
+		VirtualMachineGPUs:         vm.HardwareProfile.VirtualMachineGPUs,
```

Comment:
We have to add this field in `getVirtualmachinehardwareprofile()` as well, right?

Could you please build moccli using these changes and make sure moccli is printing the GPUs?